### DISCUSSION TITLE
TypeError: Cannot read property 'speed' of undefined !

### DISCUSSION THREAD

**Discussion by zhoubuhong** [nolimits4web/swiper#6014] [Q&A]
### Check that this is really a bug

- [x] I confirm

### Reproduction link

codesandbox.io

### Bug description

Executed at Android `swiperRef.slideTo(0);` Will report an error: "TypeError: Cannot read property 'speed' of undefined !", After the second parameter is passed in, the error disappears, that is: `swiperRef.slideTo(0, 300);`

### Swiper version

8.3.2

### Platform/Target and Browser Versions

Android 9, Android 10, Android 11, Android 11

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by zhoubuhong** [comment]
What was written before is wrong Swiper version, correct Swiper version is: 8.2.4. I'm very sorry.

**Reply by hendra-foo** [reply]
any update on this issue?, i'm using `useSwiper` and call `slideNext()` also getting this error
swiper version: 8.3.2

**Reply by ramikhafagi96** [reply]
this is the stack trace `Cannot read properties of undefined (reading 'speed')`  I'm getting on Sentry, any updates @nolimits4web?
```javascript
export default function slideNext(speed = this.params.speed, runCallbacks = true, internal) {
  const swiper = this;
  const {
    animating,
    enabled,
    params
...
```

**Comment by reza-ghafoori** [comment]
any solutions? I'm facing this issue on `swiper.params.speed`, too.

**Comment by ElVisPL** [comment]
I'm also receiving many errors on Sentry related to this

```
timeout = nextTick(() => {
              const snapToThreshold = 0.5;
              lastEventBeforeSnap = newEvent;
              recentWheelEvents.splice(0);
              swiper.slideToClosest(swiper.params.speed, true, undefined, snapToThreshold); //  <----HERE
            }, 500);
           
```
